tests/test_touringcar/test_view.py

```diff
@@ -0,0 +1,56 @@
+import geojson
+import pytest
+from django.core.exceptions import ValidationError
+from model_bakery import baker
+
+from touringcar.model import Bericht
+from touringcar.serializer import BerichtFilterSerializer, BerichtSerializer
+
+
+@pytest.fixture
+def bericht():
+    return baker.prepare(Bericht, is_live=True, startdate="2023-11-15", enddate = "2023-11-30")
+
+@pytest.fixture
+def bericht_error():
+    return baker.prepare(Bericht, is_live=True, startdate="2023-11-15", enddate = "2023-11-04")
+
+
+@pytest.mark.django_db
+def test_error_start_enddate(bericht_error):
+    with pytest.raises(ValidationError) as e:
+        bericht_error.save()
+    assert "{'enddate': ['enddate can not be before startdate.']}" in str(e.value)
+     
+@pytest.mark.django_db
+def test_serialization(bericht):
+    bericht.save()
+    serializer = BerichtSerializer(bericht)
+    # Act
+    data = serializer.data
+    # Assert
+    assert data["properties"]["is_live"] == True
```

Comment:
kan met `is` ipv `==`

Reply:
ik vind hier == beter aangezien het om een 'waarde' vergelijking gaat. `is_live` is een boolean.